src/app/content/components/ContentLink.tsx

```diff
@@ -53,8 +53,16 @@ export const ContentLink = (props: React.PropsWithChildren<Props>) => {
     ...anchorProps
   } = props;
   const {url, params} = getBookPageUrlAndParams(book, page);
-  const relativeUrl = toRelativeUrl(currentPath, url);
+  let relativeUrl = toRelativeUrl(currentPath, url);
```

Comment:
I think that maybe `toRelativeUrl` should accept the argument with search options or we should leave this as it was (as `const`) and then create `const URL = ${relativeUrl}?${options.search};`

not a big deal